embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/getter/ColumnGetterFactory.java

```diff
@@ -62,6 +62,8 @@ private ColumnGetter newColumnGetter(JdbcInputConnection con, PluginTask task, J
             return new TimestampColumnGetter(to, toType, newTimestampFormatter(option, DateColumnGetter.DEFAULT_FORMAT));
         case "decimal":
             return new BigDecimalColumnGetter(to, toType);
+        case "array":
```

Comment:
Because "array" type is specific for PostgreSQL, these lines should be in PostgreSQLColumnGetterFactory class.

Reply:
Got it. I moved to `embulk-input-postgresql`.